examples/sample-pub/publisher.js

```diff
@@ -130,7 +130,15 @@ function startFlow(flow, var_args) {
  * Current valid values are: 'showOffers', 'linkAccount', 'getEntitlements'.
  */
 function startFlowAuto() {
-  const flow = (window.location.search || '').split('?')[1] || 'demo';
+  let flow = (window.location.search || '').split('?')[1] || 'demo';
+
+  // Check for GAA param.
+  const queryParams = getQueryParams();
+  if (queryParams.gaa_at) {
```

Comment:
Is it documented somewhere else how to trigger these different flows? If so please link to it in a comment.